README.md

```diff
@@ -151,7 +154,8 @@ Custom directives can be embedded in HTML four different ways. Demonstrate **two
 
 **Your answer:**
 ```html
-
+<input ng-model="foo">
+<person>{{name}}</person>
```

Comment:
not quite... see https://github.com/ga-wdi-quizzes/week-08/blob/beckys-changes/README.md#question-9